common_package_py/common_node.py

```diff
@@ -68,10 +74,90 @@ def __heartbeat_timer_callback__(self) -> None:
         """
         
         msg = Heartbeat()
+        
         msg.sender_id = self.get_fully_qualified_name()
         msg.active = self.__node_active__
         self.__tick__ += 1
         msg.tick = self.__tick__
         msg.time_stamp = self.get_clock().now().to_msg()
+        
         self.__heartbeat_publisher__.publish(msg)
+        
         self.get_logger().debug(f"CommonNode::__heartbeat_timer_callback__: Published heartbeat message with sender_id: {msg.sender_id}, tick: {msg.tick}, active: {msg.active}")
+
+    def _job_finished_custom_(self, error_code: int, payload: dict) -> None:

```

Comment:
Wieso machst du das nicht mit default arguments, die man als keyword übergibt.
```suggestion
    def _job_finished_(self, *, error_code: int = EXIT_SUCCESS, payload: dict = {}, error_message: str = "") -> None:
```

Reply:
Ich wollte es nicht zu komplex in der Verwendung machen. Das den Usern zu erklären und welche Kombinationen erlaubt sind ist mmN zu komplex.